src/test/resources/dev.properties

```diff
@@ -0,0 +1,9 @@
+testdata.instance.number=4
+testdata.instance.software=FREE
+testdata.instance.vmClass=REGULAR
+testdata.instance.type=STANDART8
+testdata.instance.gpuNumber=ONE
+testdata.instance.gpuType=TESLAK80
+testdata.instance.ssd=SSD2
+testdata.instance.location=FRANKFURT
+testdata.instance.usage=YEAR1
```

Comment:
Add new lines to every file to avoid such notifications. 
![image](https://user-images.githubusercontent.com/42807344/83689734-3baeb800-a5f8-11ea-82f8-162c0033ca59.png)